parser/src/main/java/parser/IPageUrlParser.java

```diff
@@ -9,13 +10,14 @@
 import java.util.Optional;
 import java.util.stream.Collectors;
 
-/**
- * Created by Grzesiek on 2016-08-28.
- */
-@Log4j2
-public abstract class UrlParser {
+public interface IPageUrlParser {
 
-    protected List<String> getLinksList(Optional<Document> document, String cssQuery) {
+    Logger log = LogManager.getRootLogger();
```

Comment:
I am not sure, why in some classes you use Lombok and in some you don't for logging?